model-optimizer/extensions/front/tf/pad_tf_to_pad.py

```diff
@@ -34,6 +35,9 @@ def find_and_replace_pattern(self, graph: Graph):
                 # the input with fill value is an optional third input in TF
                 if not tfpad.in_port(2).disconnected():
                     tfpad.in_port(2).get_connection().set_destination(new_pad.in_port(3))
+                else:
```

Comment:
Please, fix the `unit_tests/extensions/front/tf/pad_tf_to_pad_test.py` test. Now the unit test fails because of new changes

Reply:
fixed test